spec/factories/recording_factory.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :recording do
+    room
```

Comment:
Does this create a room using room factory? Then assigns the recording to that room?

Reply:
Yep